.eslintrc

```diff
@@ -4,6 +4,7 @@
   ],
   "parser": "babel-eslint",
   "rules": {
-    "no-underscore-dangle": 0
+    "no-underscore-dangle": 0,
+    "linebreak-style": 0
```

Comment:
Have you considered adding it to our [company's eslint config](https://github.com/Travix-International/eslint-config-travix/)?

Reply:
I think it makes sense to disable it by default in `eslint-config-travix` and rather opt into it on a per project basis if necessary for some reason.

Reply:
[PR for making the change in company's ESLint config](https://github.com/Travix-International/eslint-config-travix/pull/22)